src/client/entity-editor/common/sort-name-field.js

```diff
@@ -148,9 +158,12 @@ function SortNameField({
 	const guessButton =
 		<Button bsStyle="primary" onClick={handleGuessClick}>Guess</Button>;
 
+	const copyButton =
+		<Button bsStyle="success" onClick={handleCopyClick}>Copy</Button>;
```

Comment:
I see no reason for this button's color to be different from the other one.
Button colors have specific meaning rather than being only decorative.

If you are worried same-color buttons would be an issue because they look like one button, you'll have to find another solution.
Unfortunately we're using Bootstrap 3 which is lacking a few nice options like outline buttons from v4 that could be a good solution.